css/style.css

```diff
@@ -0,0 +1,19 @@
+
+#title {
+    text-align: center;
+}
+
+#image {
+    max-width:100%;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+ }
+ 
+ li {
+     list-style-type: disc;
+ }
+
+ #tribute-info {
+     align-content: center;
+ }
```

Comment:
I suggest you use style linters to give you an orderliness when writing your CSS codes, so your codes look cleaner and easy to maintain 😉